src/lib/db.c

```diff
@@ -408,21 +406,6 @@ CK_RV db_get_tokens(token **tok, size_t *len) {
         cnt++;
     }
 
-    /* if their was no unitialized token in the db, add it */
```

Comment:
How do we know an unitialized token was added to the list? We're assuming FAPI will provide one? But what if FAPI fails?

Reply:
Not fapi, but the backend.c file will provide the empty token. Could also be the token.c or slot.c but I put it into backend.c